### ISSUE TITLE
Need Help

### ISSUE THREAD

**Issue by hxnnxs5073** [bablokb/autobackup-service#4]
Hello, I wanted to set up an auto backup system on a Raspberry Pi or a general Linux where when I plug the USB stick into the Raspberry Pi it automatically saves the data to a server path. Can this be done with this script?

**Comment by bablokb** [comment]
Yes. Please read the toplevel readme. It states: "This little project enables automatic backups triggered by a plugin of an USB backup device."
